packages/carbon-graphs/src/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -683,7 +683,7 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
   const pairedBoxGroup = d3.selectAll(`.${styles.pairedBoxGroup}`);
   pairedBoxGroup.each(function () {
     const clipPath = d3.select(this).attr('clip-path');
-    if (clipPath === pairedBoxGroupClipPath) {
+    if (clipPath.replace(/['"]+/g, '') === pairedBoxGroupClipPath) {
```

Comment:
Does this fix the pair results issue? Can you add a comment about why we are doing it this way?

Reply:
yes veera and comment added [here](https://github.com/cerner/terra-graphs/pull/30/commits/591d01d03d810df52c8b3f21060fec87d4cc81c3#diff-d04c4cf6f73329ea29bf2a081e4f7b5d5ddf52580b58d6ce58bf9c7b2728fb1eR686).